load-responsively.js

```diff
@@ -32,9 +32,14 @@
       $container.css({ opacity: 0, transition: 'opacity 1s ease-in-out' });
       $this.css({position: 'absolute', top: 0, left: 0, width: '100%'});
 
-      $image.on('load', function() {
+      if ($image.width !== undefined && $image.width !== 0) {
```

Comment:
What if you triggered the load event on the image if it's already loaded?
Something like:

```
if ($image.width !== undefined && $image.width !== 0) {
  $image.trigger('load');
}
```

If that works I think it would be better.